### PR TITLE
Newbranch

### PR DESCRIPTION
**Pull request by DavidPerezS**
3 exercises solving leap year and Sebastian´s issues.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (salorrego) [inline .gitignore]: EOF

File: exercise_2/index.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+const axios = require('axios');
+const { get } = require('prompt');
```
Comment (Thormod) [inline exercise_2/index.js]: Remove unused module.

File: leapyear.js [hunk 3]
```diff
@@ -1,16 +1,13 @@
-
 const prompt = require('prompt-sync')();
```
Comment (Thormod) [inline leapyear.js]: Let's create a folder and name it `exercise_0`

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+*.cab
+*.msi
+*.msix
+*.msm
+*.msp
+
+# Windows shortcuts
+*.lnk
+
+# End of https://www.toptal.com/developers/gitignore/api/node,visualstudiocode,windows
```
Comment (salorrego) [inline .gitignore]: please add EOF (a blank line at the end of the file so you can read it easily at the end)

File: exercise_3/index.js [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    console.error(error);
+  }
+};
+
+const searchJoke = async () => {
+  try {
+    searchparam = prompt('Input search term for jokes: ');
+    console.log(`Input search parameter was: ${searchparam}
+     `);
+    await getJoke()
```
Comment (Thormod) [inline exercise_3/index.js]:
```suggestion
    await getJoke(searchparam)
```

File: exercise_3/index.js [hunk 6]
```diff
@@ -0,0 +1,56 @@
+const axios = require('axios');
+const prompt = require('prompt-sync')();
+const fs = require('fs');
+
+let searchparam = '';
+
+const getJoke = async () => {
```
Comment (Thormod) [inline exercise_3/index.js]:
```suggestion
const getJoke = async (searchParam) => {
```

File: exercise_3/index.js [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+  const header = {
+    headers: { Accept: 'text/plain' },
+  };
+  try {
+    return await axios.get(
+      `https://icanhazdadjoke.com/search?term=${searchparam}`,
```
Comment (Thormod) [inline exercise_3/index.js]:
```suggestion
      `https://icanhazdadjoke.com/search?term=${searchParam}`,
```

File: exercise_3/index.js [hunk 8]
```diff
@@ -0,0 +1,56 @@
+const axios = require('axios');
+const prompt = require('prompt-sync')();
+const fs = require('fs');
+
+let searchparam = '';
```
Comment (Thormod) [inline exercise_3/index.js]: delete this variable in order to use functions parameters

**Review by Thormod — CHANGES_REQUESTED** [review]

**Review by Thormod — DISMISSED** [review]

**Review by Thormod — APPROVED** [review]

**Pull request merged**